source/base/parameter_handler.cc

```diff
@@ -1610,6 +1634,55 @@ ParameterHandler::declare_entry (const std::string           &entry,
 
 
 
+void
+ParameterHandler::declare_alias(const std::string &existing_entry_name,
+                                const std::string &alias_name,
+                                const bool         alias_is_deprecated)
+{
+  // see if there is anything to refer to already
+  Assert (entries->get_optional<std::string>(get_current_full_path(existing_entry_name)),
+          ExcMessage ("You are trying to declare an alias entry <"
+                      + alias_name +
+                      "> that references an entry <"
+                      + existing_entry_name +
+                      ">, but the latter does not exist."));
+
+  // now also make sure that if the alias has already been
+  // declared, that it is also an alias and refers to the same
+  // entry
+  if (entries->get_optional<std::string>(get_current_full_path(alias_name)))
```

Comment:
Dunno. We allow declaring the same parameter entry multiple times. I thought I'd just mirror existing behavior. Feel strongly about this?